Example for new versions?

I really like this example and I am setting up a [system](https://github.com/digitaldeacon/memberhive2) similar to this.

Angular, Angular-Cli and Typescript have changed somewhat since your example setup. I am trying to use the latest stuff (including ngrx/platform) but getting issues (either compiling or installing or importing the core).

Simply ngc with your tsconfig gives compiler error

I have used yarn link, but there are issue with the typing files (js in ts not allowed).

This is the config that got me relatively close to working:
```
{
  "compilerOptions": {
    "baseUrl": "",
    "declaration": false,
    "target": "es5",
    "module": "es2015",
    "moduleResolution": "node",
    "sourceMap": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": ["es2015", "dom"],
    "outDir": "./dist",
    "typeRoots": [
      "../node_modules/@types"
    ],
    "noImplicitAny": false,
    "suppressImplicitAnyIndexErrors": true,
    "paths": {
      "@ngrx/platform": ["./node_modules/@ngrx/platform"]
    }
  },
  "exclude": [
    "node_modules",
    "dist",
    "scripts"
  ],
  "angularCompilerOptions": {
    "genDir": "aot",
    "skipMetadataEmit" : true
  }
}
```

Do you have a working example for all the latest dependencies?

Hi Benoit. Thanks for the upgrade. It looks good so far. There is an issue with the mobile but that may be my env (see error below). 

I wonder, did you try to link the packages as well? It works for me when I `yarn link` the core to web and then remove the node_modules/ folder. After that ng serve works beautifully. It is the issue as described in the link I posted earlier.

Error for `ionic serve`:

```
Error: Cannot find module 'reflect-metadata'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Applications/MAMP/htdocs/hacktisch/ang2/ngrx-demo-apps/mobile/node_modules/@ionic/app-scripts/dist/aot/aot-compiler.js:4:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

--- a/README.md
+++ b/README.md
@@ -46,7 +46,8 @@ yarn
 # Or npm install
 
 # Compile typescript into dist
-npm run build
+yarn build
+# Or npm run build
 
 yarn link
 # Or npm pack

--- a/mobile/package.json
+++ b/mobile/package.json
@@ -9,6 +9,7 @@
     "ionic:serve": "ionic-app-scripts serve"
   },
   "dependencies": {
+    "@angular/animations": "4.0.1",
     "@angular/common": "4.0.1",
     "@angular/compiler": "4.0.1",
     "@angular/compiler-cli": "4.0.1",
@@ -26,7 +27,7 @@
     "@ngrx/effects": "2.0.2",
     "@ngrx/store": "2.2.1",
     "@ngrx/store-devtools": "3.2.4",
-    "ionic-angular": "2.3.0",
+    "ionic-angular": "3.0.0",
     "ionicons": "3.0.0",
     "ngrx-store-freeze": "0.1.9",
     "ngrx-store-logger": "0.1.7",
@@ -35,6 +36,7 @@
     "zone.js": "0.8.5"
   },
   "devDependencies": {
+    "@angular/tsc-wrapped": "4.0.1",
     "@ionic/app-scripts": "1.2.5",
     "@ionic/cli-build-ionic-angular": "0.0.4",
     "@ionic/cli-plugin-cordova": "0.0.10",

